go/libraries/utils/argparser/args_test.go

```diff
@@ -112,23 +112,23 @@ func TestParsing(t *testing.T) {
 			expectedArgs: []string{"value"},
 		},
 		{
-			name:         "-mvalue",
+			name:         "-m value",
 			options:      []*Option{forceOpt, messageOpt},
-			args:         []string{"b", "-mvalue", "c"},
+			args:         []string{"b", "-m value", "c"},
 			expectedOpts: map[string]string{"message": "value"},
 			expectedArgs: []string{"b", "c"},
 		},
 		{
 			name:         "--messagevalue",
 			options:      []*Option{forceOpt, messageOpt},
-			args:         []string{"b", "-messagevalue", "c"},
+			args:         []string{"b", "--messagevalue", "c"},
```

Comment:
Disallowing long form args to have value suffixes seems like a good change. This would also be consistent with Git's arg parsing behavior (e.g. `git init --initial-branchfoo` does **not** work, but `git init --initial-branch foo` does).